ndk/src/user/index.ts

```diff
@@ -380,6 +380,7 @@ export class NDKUser {
         for (const follow of currentFollowList) {
             if (follow.pubkey !== user.pubkey) {
                 newUserFollowList.add(follow);
+            } else {
                 foundUser = true;
```

Comment:
Thanks for the PR! 

The changes look good but it looks like we were never using the `foundUser` variable? I'm guessing the reason for that initially was so that we could skip creating a new event if the unfollowed user wasn't actually in the current contact list? 

Any chance you want to add that basic skip logic to this PR so we don't try and publish a new event when it's unnecessary?

Reply:
The variable is in use in line **388**, it's possible that GitHub collapsed that line or something, so it looked like it wasn't there:

https://github.com/nostr-dev-kit/ndk/pull/230/files#diff-cbed2525c06e1a4ce1190d74f12c6a2883780edf9f8fa70e237330a116711740R388

![image](https://github.com/nostr-dev-kit/ndk/assets/108793/64848fd9-c95a-4c6e-abd8-916106be2384)

Reply:
haha, sneaky little bastard.